src/cacheReader/exporters/GLTFExporter.js

```diff
@@ -561,17 +562,46 @@ export default class GLTFExporter {
             let v1 = this.remappedVertices[def.faceVertexIndices1[faceId]][lookupKey].idx;
             let v2 = this.remappedVertices[def.faceVertexIndices2[faceId]][lookupKey].idx;
             let v3 = this.remappedVertices[def.faceVertexIndices3[faceId]][lookupKey].idx;
-            alphaUvs[v1] = [paletteIndex / numUniqueColors + half, 0.33];
-            alphaUvs[v2] = [paletteIndex / numUniqueColors + half, 0.5];
-            alphaUvs[v3] = [paletteIndex / numUniqueColors + half, 0.66];
+            this.alphaUvs[v1] = [paletteIndex / numUniqueColors + half, 0.33];
+            this.alphaUvs[v2] = [paletteIndex / numUniqueColors + half, 0.5];
+            this.alphaUvs[v3] = [paletteIndex / numUniqueColors + half, 0.66];
         }
-        this.file.addColors(normalUvs, colorPalettePng, 0);
+    }
+
+    constructFile() {
```

Comment:
this stuff is now put into the GLTF at the end, it's not necessary but I think it's a nicer way to ensure the GLTF buffers, accessors etc are in the right order